app/resources/games/Heretic2/heretic2.fgd

```diff
@@ -1745,14 +1745,14 @@ Do not have teleport points, or start spots of any kind inside or inbetween this
 		2 : "Start off" : 0
 	]
 	style(integer) : "Style"
-	attenuation(choices) : "Attenuation" : 0 = 
+	attenuation(choices) : "Attenuation" : 0 =
 	[
 		0 : "No attenuation"
 		1 : "Normal"
 		2 : "High"
 		3 : "Max"
 	]
-	volume(integer) : "Volume" : 0.5
+	volume(string) : "Volume" : "0.5"
```

Comment:
this could be `(float)` right? 

The example from: https://developer.valvesoftware.com/wiki/FGD is:

```
foo(float) "Floating point number" : "100.7" : "Decimal points = fun"
```

Reply:
Oh..  Yes you are correct. Fixed my fix.